apps/moderatorfeedback/templates/meinberlin_moderatorfeedback/includes/moderatorfeedback.html

```diff
@@ -1,9 +1,7 @@
-{% if item.get_moderator_feedback %}
-    {% with item.get_moderator_feedback as moderator_feedback %}
-        {% if moderator_feedback.value %}
-            <span class="label label--big label--{{ moderator_feedback.value_class }} {{ cls }}">
-                {{ moderator_feedback.display }}
-            </span>
-        {% endif %}
-    {% endwith %}
-{% endif %}
+{% with item.get_moderator_feedback as moderator_feedback %}
+    {% if moderator_feedback.value %}
+        <span class="label label--big label--{{ moderator_feedback.value_class }} {{ cls }}">
+            {{ moderator_feedback.display }}
+        </span>
+    {% endif %}
+{% endwith %}
```

Comment:
The if was not actually redundant: It checks whether the function `item.get_moderator_feedback()` exists. @kleingeist what do you think?